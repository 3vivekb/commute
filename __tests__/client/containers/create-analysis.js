/* global describe, expect, it */

import {mount} from 'enzyme'
import {mountToJson} from 'enzyme-to-json'
import React from 'react'
import {Provider} from 'react-redux'

import {makeMockStore, mockStores} from '../../test-utils/mock-data.js'

import CreateAnalysis from '../../../client/containers/create-analysis'

const mockStore = makeMockStore(mockStores.init)

describe('Container > CreateAnalysis', () => {
  it('Create Analysis View loads', () => {
    // mount component
    const tree = mount(
      <Provider store={mockStore}>
        <CreateAnalysis />
      </Provider>
    )
    expect(mountToJson(tree.find(CreateAnalysis))).toMatchSnapshot()
  })

  /* it('Create analysis', () => {
    throw new Error('unimplemented')
  }) */
})
