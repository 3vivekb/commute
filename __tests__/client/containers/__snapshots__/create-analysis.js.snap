exports[`Container > CreateAnalysis Create Analysis View loads 1`] = `
<Connect(CreateAnalysis)>
  <CreateAnalysis>
    <h1>
      Create Analysis
    </h1>
  </CreateAnalysis>
</Connect(CreateAnalysis)>
`;
